ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/remote/AbstractConnectorExecutor.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.ml.engine.algorithms.remote;
+
+import org.opensearch.ml.common.connector.Connector;
+import org.opensearch.ml.common.connector.ConnectorHttpClientConfig;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+public abstract class AbstractConnectorExecutor implements RemoteConnectorExecutor {
+    private Integer maxConnections;
```

Comment:
Can we use `ConnectorHttpClientConfig` here ? So we don't need to add more and more fields to this class

Reply:
Good idea, addressed that.